txnkv/transaction/2pc.go

```diff
@@ -1903,6 +1905,7 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 		}
 	}
 	atomic.StoreUint64(&c.commitTS, commitTS)
+	c.updateStoreCommitInfo()
```

Comment:
The transaction is not ensured to be committed successfully here, for example, if 2PC is used.